backend/src/schema/resolvers/searches.js

```diff
@@ -20,7 +20,6 @@ export default {
       AND NOT (
         author.deleted = true OR author.disabled = true
         OR resource.deleted = true OR resource.disabled = true
-        OR (:User { id: $thisUserId })-[:BLOCKED]-(author)
```

Comment:
as discussed in our pairing session @roschaefer ... this was unintended after merging in changes from `master`
I found out why it was still passing the cypress tests and will push up the code...
turns out we had some issues with copy :spaghetti: in the `Search User` PR where we didn't properly rename variables